index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    <title>Youtube</title>
+</head>
+<body>
+    <nav class="navbar navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand " href="#">
+            <img class="logo"  src="images/youtube_logo.PNG" alt="" width="110" height="51" class="d-inline-block align-text-top">
+          </a>
+        </div>
+      </nav>
```

Comment:
Great Job  💪  of using the nav tag but should have wrapped your nav tag in the header tag, This is the best practice ❤️  for a page to have a header tag as per HTML semantics, and typically nav tag is wrapped in the header tag like this: 
<header>

  <nav>

  </nav>

</header>